### ISSUE TITLE
"Good For:" settings

### ISSUE THREAD

**Issue by GrantSP** [Thalassicus/cep-bnw#73]
If you disable the "Good For:" setting for buildings the display shows the database name for the selected building or Wonder. ie. BUILDING_COLOSSUS

It should act the same way as it does for units and just remove that display panel.

**Comment by Thalassicus** [comment]
Could you be more specific? Is this the situation:
1. Set SHOW_GOOD_FOR_BUILDINGS to 0 in Cat_Options.sql.
2. The entire building tooltip shows only the building type, with nothing else on the tooltip.

**Comment by GrantSP** [comment]
Yes. It shows the building type in the panel that showed the Good For info.
